tests/integration-service/integration.go

```diff
@@ -2,6 +2,7 @@ package integration
 
 import (
 	"fmt"
+	"github.com/redhat-appstudio/release-service/metadata"
```

Comment:
Shouldn't be this rather operator-toolkit import?

Reply:
Yep it should, fixed it here. The rest of e2e tests seem to use release-service's metadata, but won't touch that in this PR.